sessionctx/variable/session.go

```diff
@@ -127,6 +127,10 @@ type SessionVars struct {
 
 	// StmtCtx holds variables for current executing statement.
 	StmtCtx *StatementContext
+
+	// CurrValues is used to record current ValuesExpr's values.
+	// See http://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_values
+	CurrValues interface{}
```

Comment:
Rename to `CurrInsertRow` is better.
`CurrInsertRow` just contains the value of `ValueExpr`,  not the values of the `ValueExpr`.